<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QDialog" name="NewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New game</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelGameFile">
     <property name="text">
      <string>Game file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="editGameFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowseGame">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelResourceType">
     <property name="text">
      <string>Resources</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboResourceType">
       <item>
        <property name="text">
         <string>FileSystem</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>URL</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editResourceLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowseResources">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowseGame</sender>
   <signal>pressed()</signal>
   <receiver>NewGame</receiver>
   <slot>browseGameFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowseResources</sender>
   <signal>pressed()</signal>
   <receiver>NewGame</receiver>
   <slot>browseResources()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseGameFile()</slot>
  <slot>browseResources()</slot>
 </slots>
</ui>
